alcs-api/src/common/automapper/reconsideration.automapper.profile.ts

```diff
@@ -12,7 +22,67 @@ export class ReconsiderationProfile extends AutomapperProfile {
 
   override get profile() {
     return (mapper) => {
-      createMap(mapper, ReconsiderationType, ReconsiderationTypeDto);
+      createMap(mapper, ApplicationReconsiderationType, ReconsiderationTypeDto);
+
+      createMap(
+        mapper,
+        ApplicationReconsiderationCreateDto,
+        ApplicationReconsideration,
+        forMember(
+          (r) => r.submittedDate,
+          mapFrom((rd) => new Date(rd.submittedDate)),
+        ),
+      );
+
+      createMap(
```

Comment:
The intention was not to perform mapping manually since we have automapper. Agreed to have automapper only from entity -> dto since typescript does not guarantee that dto -> entity will be full/correct mapping